tests/interfaces/default_qubit_2_integration/test_autograd_qnode_default_qubit_2.py

```diff
@@ -553,6 +553,9 @@ def test_probability_differentiation(
     ):
         """Tests correct output shape and evaluation for a tape
         with a single prob output"""
+        if "lightning" in getattr(dev, "short_name", ""):
+            pytest.xfail("lightning does not support measuring probabilities with adjoint.")
```

Comment:
fyi given the shoutout this morning - I don't think this will ever xpass because you use `pytest.xfail` instead of `pytest.mark.xfail`, but that's all we've got for mid-test xfails

Reply:
You're right. Well, that's disappointing. Oh well.